_pages/cases.md

```diff
@@ -14,6 +14,7 @@ redirect_from:
   - /enforce.htm
   - /enforce_activities/
 sidenav: false
+compact: true
```

Comment:
Let's make sure this is documented in our wiki: https://github.com/usdoj-crt/beta-ada/wiki/Front-matter

Reply:
nice, thanks!